resources/definitions/ultimaker.def.json

```
@@ -40,6 +40,24 @@
         {
             "value": false,
             "enabled": false
+        },
+        "line_width": {
+            "value": "machine_nozzle_size"
+        },
+        "infill_before_walls": {
+            "value": "False"
+        },
+        "retraction_combing": {
+            "value": "'no_outer_surfaces'"
+        },
+        "skin_monotonic" : {
+            "value": true
+        },
+        "speed_equalize_flow_width_factor": {
+            "value": "1.1"
```

Is now a percentage should be:
```suggestion
            "value": "110.0"
```